pkg/subnet/local.go

```diff
@@ -77,6 +86,121 @@ func (d *LocalDeployer) DeployToLocalNetwork(chain string, chainGenesis []byte,
 	return d.doDeploy(chain, chainGenesis, genesisPath)
 }
 
+func getAssetID(wallet primary.Wallet, tokenName string, tokenSymbol string, maxSupply uint64) (ids.ID, error) {
+	xWallet := wallet.X()
+	owner := &secp256k1fx.OutputOwners{
+		Threshold: 1,
+		Addrs: []ids.ShortID{
+			genesis.EWOQKey.PublicKey().Address(),
```

Comment:
assumes ANR keys. probably should add this functionality to ANR as a separate issue

Reply:
https://github.com/ava-labs/avalanche-cli/issues/738